bindings/python/pymongocrypt/binding.py

```diff
@@ -1443,6 +1435,14 @@ def __getattr__(self, name):
     lib = _Library(exc)
 else:
     # Check the libmongocrypt version when the library is found.
+    def _parse_version(version_str):
+        # libmongocrypt versions start with x.y.z and have an optional
+        # hypen (e.g. 1.8.0 or 1.8.5-alpha1)
+        if '-' in version_str:
+            version_str = version_str[version_str.index('-')]
```

Comment:
version_str will always be `"-"` after this line.

Reply:
Fixed